distribution/docs/src/main/resources/content/_installing/file-system-access.adoc

```diff
@@ -29,6 +29,12 @@ The system administrator must restrict certain directories to ensure that the ap
 For example the `${branding}_USER` should have read-only access to `<${branding}_HOME>`, except for the sub-directories `etc`, `data`, `solr` and `instances`.
 ====
 
+.File Location Security Warning
+[WARNING]
+====
+To avoid file permission problems, it is not recommended to install ${branding} in the user's home directory or a subdirectory of home.
```

Comment:
❗️ It ain't a file permission problems but the fact that all files would automatically be accessible and modifiable by all through ddf.